src/main/java/com/sfuentes/Farm/Hen.java

```diff
@@ -0,0 +1,25 @@
+package com.sfuentes.Farm;
+
+public class Hen {
+  double henWeight;
+  double henHeight;
+  int numberOfEggs;
```

Comment:
All of these properties should be private.
As a rule of thumb properties should always remain private.

Reply:
fixe.